app/src/main/java/com/yeonae/chamelezone/view/review/ReviewCreateActivity.kt

```diff
@@ -167,18 +164,18 @@ class ReviewCreateActivity :
         image_container.removeAllViews()
 
         uris.forEachIndexed { _, uri ->
-            val rl = LayoutInflater.from(this).inflate(
+            val cl = LayoutInflater.from(this).inflate(
                 R.layout.slider_item_image,
                 image_container,
                 false
             ) as ConstraintLayout
```

Comment:
ConstraintLayout의 속성을 사용할것이 아니니 이렇게 사용하는게 좋을 수 있을 것 같아요.
```suggestion
            ) as ViewGroup
```

Reply:
수정했습니다!
cea1a7e